packages/@uppy/provider-views/src/Breadcrumbs.js

```diff
@@ -20,12 +20,11 @@ module.exports = (props) => {
       <div class="uppy-Provider-breadcrumbsIcon">{props.breadcrumbsIcon}</div>
       {
         props.directories.map((directory, i) => (
-          <Breadcrumb
```

Comment:
@goto-bus-stop Breadcrumb seemed to be broken from [this change](https://github.com/transloadit/uppy/commit/733de659da17ee77afba9882633477fb7cd778dd#diff-3a8b11d5620b3d342e81e97925b37113L18). Is there a different way you would have fixed this?

Reply:
Ah that did break it…let's just keep doing what you're doing here then!